packages/loopring_v3/circuit/Utils/Data.h

```diff
@@ -14,14 +16,130 @@ using json = nlohmann::json;
 namespace Loopring
 {
 
-enum class BlockType
-{
-    RingSettlement = 0,
+static auto dummySpotTrade = R"({
+    "fFillS_A": 0,
```

Comment:
In production, do we also use JSON to represent the actual data? I would strongly suggest that we use ProtocolBuf for serialization and deserialization - it's much faster and the data is much smaller.

Reply:
As far as I know we do. Serialization/deserialization (at least for the circuit part, I don't know for the relayer) times are of no concern because it has no impact on overall performance, so whatever is easiest over multiple languages. Everything already uses JSON now so probably doesn't make sense to switch at this point?

Reply:
Let me confirm with the relayer team whether serialization or deserialization is a performance issue, but I guess we can always add one more serialization/deserialization support later on without removing the JSON-based one.

@hosschao can you follow up on this?